examples/models/llama2/export_llama_lib.py

```diff
@@ -406,7 +407,11 @@ def build_args_parser() -> argparse.ArgumentParser:
         help="Use cProfile to profile model export. Results saved to profile_path as a html file.",
     )
     parser.add_argument(
-        "-G", "--group_size", default=None, help="group_size for weight quantization"
+        "-G",
+        "--group_size",
+        type=int,
+        default=256,
```

Comment:
since group_size doesn't make sense for other settings like fp, maybe default to None as well

Reply:
We should probably do the same for the GPTQ specific args then (calibration_*)

I'll change this PR to update all of them and the lower one to just plumbing and keeping None